src/App.js

```diff
@@ -3,18 +3,25 @@ import './App.css';
 import timelineData from './data/timeline.json';
 
 import Timeline from './components/Timeline';
+// import TimelineEvent from './components/TimelineEvent'; //temp
+
 
 class App extends Component {
+
   render() {
     console.log(timelineData);
+    const eventObjects = timelineData.events.map((event, i) => {
```

Comment:
If you take a look at the JSON, you will see that there is an Events key which is the array of statuses. Both seem to work. :)